templates/sectionsummary.html

```diff
@@ -1 +1,60 @@
-{% extends 'summary.html' %}
+{% extends 'layouts/_questionnaire.html' %}
+{% from "components/panel/_macro.njk" import onsPanel %}
+{% from "components/button/_macro.njk" import onsButton %}
+
+{% import 'macros/helpers.html' as helpers %}
+
+{% set summary_page_title = _("Summary") + " - " + survey_title %}
```

Comment:
Should this should use the section summary title?

Reply:
As in `content.summary.title`?

Reply:
I'm not sure this is even needed here actually. I'll remove it and test

Reply:
It isn't required